src/Windows/Avalonia.Win32/WindowImpl.cs

```diff
@@ -18,7 +18,7 @@
 
 namespace Avalonia.Win32
 {
-    public class WindowImpl : IWindowImpl
+    class WindowImpl : IWindowImpl
```

Comment:
I think this one should probably actually stay public: I was thinking that in future we might want to expose platform-specific functionality via allowing casting of the `Window.PlatformImpl` into ` WindowImpl`.